codexctl/updates.py

```
@@ -272,0 +1,372 @@
+            tuple: A tuple containing the version ids for the remarkablepp, remarkable2, remarkable1, toltec version and external provider (in that order)
+        """
+
+        if os.path.exists("data/version-ids.json"):
+            file_location = "data/version-ids.json"
+
+            self.logger.debug("Found version-ids at data/version-ids.json")
+
+        else:
+            if os.name == "nt":  # Windows
+                folder_location = os.getenv("APPDATA") + "/codexctl"
+            elif os.name == "posix" or "darwin":  # Linux or MacOS
```

Ruff (0.8.0)

Use `True` instead of `... or True`

Replace with `True`

(SIM222)